Branch-SDK/src/io/branch/referral/Branch.java

```diff
@@ -291,8 +291,11 @@
 
     private static boolean isLogging_ = false;
 
-    private static boolean checkInstallReferrer_ = false;
-    private static long PLAYSTORE_REFERRAL_FETCH_WAIT_FOR = 5000;
+    // Branch will wait for 1.5 sec to capture the install referrer inorder to provide
+    // accurate attribution . This will be only on the first time user open the app.
+    // @see to skip this please call {@link #disablePlayStoreReferrer() }
+    private static boolean checkInstallReferrer_ = true;
```

Comment:
my only concern with this PR is setting this default to true, what are your thoughts @sojanpr